sources/dicom/test_dicom/test_web_client.py

```diff
@@ -0,0 +1,18 @@
+import os
+import sys
+
+import pytest
+
+pytestmark = [
+    pytest.mark.skipif(sys.version_info < (3, 8), reason='requires python3.8 or higher'),
+]
+
+
+@pytest.mark.girder_client
```

Comment:
I think we need both the `@pytest.mark.girder` and `@pytest.mark.girder_client` to be consistent with other marks.

Reply:
And, I think the linter will demand these are `@pytest.mark.girder()` and `@pytest.mark.girder_client()`.

Reply:
Fixed